src/i18n/es.js

```diff
@@ -59,22 +59,38 @@ const messages_es = {
     volume_do_not_know: 'No lo sé',
   },
   big_query: {
-    add_google_account: `Agregar cuenta de Google`,
+    add_google_account: `Agregar cuenta de Google.`,
     add_permission_google_account: `Agregar permiso para cuentas de Google.`,
     free_alt_image: `big query`,
     free_btn_redirect: `Conoce nuestro plan Plus`,
-    free_text_data_studio_MD: `Aprende a configurar nuestro tablero de [Data Studio en unos pocos pasos](https://datastudio.google.com/overview),
-    o accede a BigQuery mediante otra herramienta de visualización de datos como Tableau.`,
-    free_text_strong: `¡Potencia tu cuenta con reportes de BigQuery!`,
-    free_text_summary: `Integra Google BigQuery para analizar reportes detallados sobre tus campañas,tus suscriptores y otros datos de tu cuenta en un tablero personalizable.`,
-    free_title: `Big Query`,
-    free_ul_item_filter: `Podras filtrar los reportes en base a todas tus campañas o seleccionar solo algunas en particular.`,
-    free_ul_item_insights: `Obtén valiosos insights sobre tu cuenta`,
-    plus_button_add: `+`,
-    plus_button_remove: `-`,
-    plus_error_message_email_exists: `El correo ingresado ya se encuentra configurado`,
-    plus_loading_message: `Cargando ...`,
+    free_text_data_studio_MD: `[Configura el tablero prediseñado de Google Data Studio en unos pocos pasos](https://developers.google.com/datastudio) para analizar reportes detallados sobre tus campañas,
+    tus suscriptores y otros datos de tu cuenta en un tablero personalizable`,
+    free_text_strong: `Potencia tus Campañas con reportes de Google Data Studio, Tableau, Power BI y otras herramientas de Data Analytics.`,
+    free_text_summary: `Accede a reportes detallados para analizar el rendimiento de tus Campañas de Email & Automation Marketing. Con esta integración podrás: `,
+    free_title: `Conecta Doppler con BigQuery`,
+    free_ul_item_filter: `Aplicar diferentes filtros para enfocar el análisis en uno o varios aspectos de tu estrategia.`,
+    free_ul_item_insights: `Trabajar desde un tablero personalizado y con vistas editables.`,
+    free_ul_item_strategy: `Obtener información valiosa para tu estrategia.`,
+    plus_beginning: `Inicio`,
+    plus_big_query: `BigQuery`,
+    plus_body_step_one_MD: `Otorgar acceso a BigQuery. [HELP](https://cloud.google.com/bigquery/docs/?hl=es-419)`,
+    plus_body_step_two_MD: `Crear reportes. [HELP](https://datastudio.google.com/overview)`,
+    plus_configuration: `Configuración`,
+    plus_control_panel: `Panel de control`,
```

Comment:
```suggestion
    plus_control_panel: `Panel de Control`,
```
![image](https://user-images.githubusercontent.com/2439363/132355506-628d4efe-9389-4f87-8e2e-4b2abb81054e.png)

Reply:
suggest accepted and applied